src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -12,10 +12,52 @@ public class HelpCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String SHOWING_HELP_MESSAGE = "Available Commands:\n"
+            + "- help: Shows details on available commands\n"
+            + "- add: Adds a new contact\n"
+            + "- delete: Removes a contact\n"
+            + "- list: Displays stored contacts";
+
+    private final String ShowingCommandMessage;
+    /**
+     * Creates an HelpCommand
+     */
+    public HelpCommand() {
+        this.ShowingCommandMessage = "";
+    }
+
+    /**
+     * Creates an HelpCommand according to the specified {@code Command}'s
+     * @param command {@code COMMAND_WORD} of the command
+     * @param usage {@code COMMAND_USAGE} of the command
+     * @param description the description of the command
+     */
+    public HelpCommand(String command, String usage, String description) {
+        String message = "Command:" + command
+                + "\nUsage:" + usage
+                + "\nDescription:" + description;
```

Comment:
```suggestion
        String message = "Usage:" + usage + "\nDescription:" + description;
```
<img width="185" alt="image" src="https://github.com/user-attachments/assets/5d4bf579-9213-476e-b734-3f800dc8ebdf" />

Command word is duplicated a few times and missing whitespace